Nitrox.Launcher/ViewModels/LaunchGameViewModel.cs

```diff
@@ -292,4 +301,40 @@ private void UpdateGamePlatform()
         GamePlatform = NitroxUser.GamePlatform?.Platform ?? Platform.NONE;
         PlatformToolTip = GamePlatform.GetAttribute<DescriptionAttribute>()?.Description ?? "";
     }
+
+    // BepInEx mod detection
+    private static bool AreBepInExModsPresent(string gameDir, out int modCount)
+    {
+        modCount = 0;
+        if (string.IsNullOrWhiteSpace(gameDir) || !Directory.Exists(gameDir))
```

Comment:
I understand why you have done this but I think for consistency could we have the if brackets here like we do in most of the code base